ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
+
+```python
+import pandas as pd
+import os.path
+```
+
+
+```python
+path_root = '/media/barnacle/projects/emp/'
+resultsFile = '/home/sjanssen/Desktop/observations.tsv'
```

Comment:
I think it already is very general, you just have to replace those two paths with whatever suits your machine. Everything else should not be hard coded.